packages/strapi-design-system/src/Textarea/TextareaInput.tsx

```diff
@@ -0,0 +1,70 @@
+import { forwardRef, MutableRefObject } from 'react';
+
+import styled from 'styled-components';
+
+import { Box, BoxProps } from '../Box';
+import { useField } from '../Field';
+import { inputFocusStyle } from '../themes/utils';
+
+interface TextareaInputBoxProps extends BoxProps<HTMLTextAreaElement> {
+  disabled?: boolean;
```

Comment:
@joshuaellis I don't understand why I have to add `disabled` here manually. It should already be defined through `HTMLTextAreaElement` no?